### PR TITLE
[FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese*

## Brief change log

*Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese*
  - *page file location: docs/monitoring/checkpoint_monitoring.zh.md*
  - *page url in documentation-web is : https://ci.apache.org/projects/flink/flink-docs-release-1.11/zh/monitoring/checkpoint_monitoring.html*

## Verifying this change

*A pure translation work .*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
Hi, @klion26
Could you help me to review this PR if you have free time?
Thank you.

File: docs/monitoring/checkpoint_monitoring.zh.md
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:74]: unaligned checkpoint 翻译成 `未对齐的 Checkpoint` 这个是否合适呢？

File: docs/monitoring/checkpoint_monitoring.zh.md
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:85]: 这个的意思能否再优化下呢？如果我没猜错的话，这里应该是 "从 barrier 在 source 创建到流到这个 task 的时间差“ 当然这个含义我需要看代码 check 下

File: docs/monitoring/checkpoint_monitoring.zh.md
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:102]: 这个 history 的 tab 链接应该是有问题的

File: docs/monitoring/checkpoint_monitoring.zh.md
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:108]: 这里的意思应该是 jm 重启后，这些信息会丢失，和上文说的一样，会重新计数

File: docs/monitoring/checkpoint_monitoring.zh.md
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:114]: streaming configuration 翻译成 `流式配置` 不确定是否合适

File: docs/monitoring/checkpoint_monitoring.zh.md
Comment (klion26) [inline docs/monitoring/checkpoint_monitoring.zh.md:121]: `Checkpoint` 的大小写建议统一，非首字母可以小写

**Review by klion26 — COMMENTED** [review]
@RocMarshal thanks for the contribution, overall the change looks good, left some minor comments. please take a look.

**Comment by RocMarshal** [comment]
@klion26 
Thank you  for your help.
I made some change based  on  your suggestions.

**Comment by RocMarshal** [comment]
@xccui

**Review by klion26 — APPROVED** [review]
@RocMarshal thanks for the update, LGTM now.
cc @wuchong  do you want to have a double-check this

**Comment by wuchong** [comment]
Feel free to merge @kl0u .

**Comment by RocMarshal** [comment]
@dianfu 
Could you help me to merge this PR if there is nothing inappropriate ?

Thank you.

**Comment by dianfu** [comment]
@RocMarshal I'd like to leave it to @klion26 who is more professional in this area and have also reviewed this PR. @klion26 Feel free to merge this PR if you think it's OK to merge. :)

**Comment by klion26** [comment]
@dianfu thanks for the review, will merge this one soon.

**Comment by klion26** [comment]
@RocMarshal When I pull your branch locally, seems there are two other commits between 296c869ceac85 and 9793b9c23786a9, I'm not sure why could this happen, could you please double-check it on your side? 

And very much appreciate it if you could squash the commit into a single one, I'm not very familiar with merge commit into master currently :)


```
275378338d (HEAD -> RE_13750, R13750) Update docs/monitoring/checkpoint_monitoring.zh.md
1ec411ddcf Update docs/monitoring/checkpoint_monitoring.zh.md
9793b9c237 Update docs/monitoring/checkpoint_monitoring.zh.md
726435e10d [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
3b9e20aafc [FLINK-17528][table] Remove ArrayData#get() API and use ElementGetter instead
debbdc66a4 [FLINK-17528][table] Remove RowData#get() API and use FieldGetter instead
296c869cea [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
```

**Comment by RocMarshal** [comment]
> @RocMarshal When I pull your branch locally, seems there are two other commits between [296c869](https://github.com/apache/flink/commit/296c869ceac85a5fd5c880a371f3bc6941f97acb) and [9793b9c](https://github.com/apache/flink/commit/9793b9c23786a99b65dbe044f84fb5d1232add20), I'm not sure why could this happen, could you please double-check it on your side?
> 
> And very much appreciate it if you could squash the commit into a single one, I'm not very familiar with merge commit into master currently :)
> 
> ```
> 275378338d (HEAD -> RE_13750, R13750) Update docs/monitoring/checkpoint_monitoring.zh.md
> 1ec411ddcf Update docs/monitoring/checkpoint_monitoring.zh.md
> 9793b9c237 Update docs/monitoring/checkpoint_monitoring.zh.md
> 726435e10d [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
> 3b9e20aafc [FLINK-17528][table] Remove ArrayData#get() API and use ElementGetter instead
> debbdc66a4 [FLINK-17528][table] Remove RowData#get() API and use FieldGetter instead
> 296c869cea [FLINK-19394][docs-zh] Translate the 'Monitoring Checkpointing' page of 'Debugging & Monitoring' into Chinese
> ```

@klion26 I'll check it as soon as possible. Thank you for your reply.

**Comment by RocMarshal** [comment]
@klion26 I made a little change on the branch information. Please take a look.

**Comment by klion26** [comment]
@RocMarshal thanks for the update, merging.

**Pull request merged**
